src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -282,4 +292,36 @@ export default App.extend({
 
     this.showChildView('tooltip', tooltipView);
   },
+  showEditActionsSidebar() {
+    const headerView = new BulkEditActionsHeaderView({
```

Comment:
is there a plan to change this here?  Seems like this needs to be the entire sidebar not just the header.  FWIW thought I think the rest of the sidebar can probably be handled within this view.. I just think maybe the naming is too narrow.

Reply:
Yes, we'll also have a view for the sidebar body with the components. Since the sidebar is a modal in this case, we need a separate header, body and (potentially) footer view, don't we? Or would it make more sense to modify the sidebar modal to work more like the custom modal where we can pass a single view?

Reply:
ah no I see it now.. didn't realize this was being passed as `headerView` to the modal.. all good.